### PR TITLE
feat(cli): Enable template support by default use hello-world

### PR DESCRIPTION
**Pull request by getspooky**
Hello,
This PR allow user to choose supported aleph template, currently we just support `hello-world` default template.

Example: 
```
aleph init hello --template=app-ui
```

I am planning to add custom template like [create-react-app](https://create-react-app.dev/docs/custom-templates)

### CODE REVIEW

**Comment by getspooky** [comment]
@ije can you pls check this PR

File: cli.ts [hunk 1]
```diff
@@ -1,18 +1,18 @@
-import { resolve } from 'https://deno.land/std@0.96.0/path/mod.ts'
-import { parse } from 'https://deno.land/std@0.96.0/flags/mod.ts'
-import { existsDir } from './shared/fs.ts'
-import log, { LevelNames } from './shared/log.ts'
-import util from './shared/util.ts'
-import { VERSION } from './version.ts'
+import { parse } from "https://deno.land/std@0.96.0/flags/mod.ts";
```
Comment (ije) [inline cli.ts:1]: pls fix the format we are following:
- single quote
- no semicolon end of the line

File: shared/constants.ts [hunk 2]
```diff
@@ -1,2 +1,3 @@
-export const minDenoVersion = '1.10.1'
-export const defaultReactVersion = '17.0.2'
+export const minDenoVersion = "1.10.1";
+export const defaultReactVersion = "17.0.2";
+export const supportedAlephTemplate = ["hello-world"];
```
Comment (ije) [inline shared/constants.ts:3]: we can check the downloaded zip that includes all the templates via dir name

File: cli.ts (hunk 1 above)
Comment (getspooky) [reply cli.ts:1]: yes , sorry i run `deno fmt`

Comment (ije) [reply cli.ts:1]: cool

File: shared/constants.ts (hunk 2 above)
Comment (getspooky) [reply shared/constants.ts:3]: @ije I can check template-status from https://github.com/alephjs/alephjs-templates, i will be very fast ?

File: cli.ts (hunk 1 above)
Comment (getspooky) [reply cli.ts:1]: @ije done. can you pls review changes.

Comment (ije) [reply cli.ts:1]: sorry i can not see the changes, still double quotes

Comment (getspooky) [reply cli.ts]: @ije fixed

**Review by crypt096 — APPROVED** [review]

File: cli.ts [hunk 3]
```diff
@@ -1,18 +1,18 @@
-import { resolve } from 'https://deno.land/std@0.96.0/path/mod.ts'
 import { parse } from 'https://deno.land/std@0.96.0/flags/mod.ts'
+import { resolve } from 'https://deno.land/std@0.96.0/path/mod.ts'
 import { existsDir } from './shared/fs.ts'
 import log, { LevelNames } from './shared/log.ts'
 import util from './shared/util.ts'
-import { VERSION } from './version.ts'
+import { VERSION } from './version.ts';
```
Comment (vincenzopalazzo) [inline cli.ts:6]: I think that here there is a `;` more

File: cli.ts [hunk 4]
```diff
@@ -1,18 +1,18 @@
...
 const commands = {
   'init': 'Create a new app',
   'dev': 'Start the app in development mode',
   'start': 'Start the app in production mode',
   'build': 'Build the app to a static site (SSG)',
   'analyze': 'Analyze the app deps',
-  'upgrade': 'Upgrade Aleph.js command'
-}
+  'upgrade': 'Upgrade Aleph.js command',
+};
```
Comment (vincenzopalazzo) [inline cli.ts:15]: Same here

File: cli.ts [hunk 5]
```diff
@@ -24,82 +24,85 @@ Usage:
...
 
 Options:
     -v, --version  Prints version number
     -h, --help     Prints help message
-`
+`;
 
 async function main() {
-  const { _: args, ...options } = parse(Deno.args)
+  const { _: args, ...options } = parse(Deno.args);
```
Comment (vincenzopalazzo) [inline cli.ts:38]: And here

File: cli.ts [hunk 6]
```diff
@@ -24,82 +24,85 @@ Usage:
...
 
   // prints aleph.js version
   if (options.v) {
-    console.log(`aleph.js v${VERSION}`)
-    Deno.exit(0)
+    console.log(`aleph.js v${VERSION}`);
+    Deno.exit(0);
```
Comment (vincenzopalazzo) [inline cli.ts:43]: Ops, other `;`

**Review by vincenzopalazzo — CHANGES_REQUESTED** [review]
LGTM, but I didn't catch the main motivation that in this PR there are some additional `;`, I think that it is important to standardize these rules in the project.

anyway, Good job I think.

**Comment by ije** [comment]
@getspooky can you pls merge the master branch? since the cli/init.ts has been moved to `commands/init.ts`

**Comment by getspooky** [comment]
@vincenzopalazzo `;` is due to [Code formatter tool](https://deno.land/manual/tools/formatter), Sorry 😁 i just fixed !

**Comment by ije** [comment]
lgtm, great job @getspooky

**Comment by getspooky** [comment]
Hi @ije I am interested to join aleph.js organization

**Comment by shadowtime2000** [comment]
@getspooky If you are interested in joining the AlephJS organization, can you please privately contact @ije instead of asking here, it's probably more fitting.

**Pull request merged**